src/main/java/com/zaxxer/hikari/pool/HikariPool.java

```diff
@@ -526,7 +526,7 @@ private void fillPool()
          addConnectionExecutor.execute(new Runnable() {
             @Override
             public void run() {
-               logPoolState("After fill ");
+               logPoolState("After fill\t");
```

Comment:
yes other two callers are from test package ..MiscTest and PostgresTest and called without parameter